server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
```

Comment:
after setting `fxaccount_url` and maybe a couple of the other URLs to development URLs early on, I thought we decided all URLs should be the prod URLs so that we do not need puppet-config changes. Maybe we should go back (in another PR) and unify the others to use prod URLs everywhere, and make corresponding changes to [local.json-dist](https://github.com/mozilla/fxa-content-server/blob/master/server/config/local.json-dist) to override the prod values.